go.mod

```diff
@@ -10,7 +10,11 @@ require (
 	cloud.google.com/go/storage v1.29.0
 	github.com/AlecAivazis/survey/v2 v2.3.6
 	github.com/ClickHouse/clickhouse-go v1.4.5
+	github.com/IBM/sarama v1.43.2
 	github.com/MakeNowJust/heredoc v1.0.0
+	github.com/alibabacloud-go/darabonba-openapi/v2 v2.0.10
+	github.com/alibabacloud-go/maxcompute-20220104 v1.4.1
```

Comment:
this can be removed, no? please `go mod tidy` again while make sure you use go version 1.20.X